src/doc/book/references-and-borrowing.md

```diff
@@ -86,7 +86,7 @@ fn main() {
         return v.iter().fold(0, |a, &b| a + b);
     }
     // Borrow two vectors and sum them.
-    // This kind of borrowing does not allow mutation to the borrowed.
+    // This kind of borrowing does not allow mutation to the borrowed reference.
```

Comment:
"the borrowed" refers to the vector (or vectors), not to the reference. What could be said is "does not allow mutation *through* the borrowed reference".